src/components/SelectionList/UserListItem.js

```diff
@@ -5,18 +5,47 @@ import lodashGet from 'lodash/get';
 import PressableWithFeedback from '../Pressable/PressableWithFeedback';
 import styles from '../../styles/styles';
 import Text from '../Text';
-import {checkboxListItemPropTypes} from './selectionListPropTypes';
+import {userListItemPropTypes} from './selectionListPropTypes';
 import Avatar from '../Avatar';
 import OfflineWithFeedback from '../OfflineWithFeedback';
 import CONST from '../../CONST';
 import * as StyleUtils from '../../styles/StyleUtils';
 import Icon from '../Icon';
 import * as Expensicons from '../Icon/Expensicons';
 import themeColors from '../../styles/themes/default';
+import Tooltip from '../Tooltip';
+import UserDetailsTooltip from '../UserDetailsTooltip';
 
-function CheckboxListItem({item, isFocused = false, onSelectRow, onDismissError = () => {}}) {
+function UserListItem({item, isFocused = false, showTooltip = false, onSelectRow, onDismissError = () => {}}) {
```

Comment:
Actually, I think we should make it required and without a default value, because we always have to explicitly pass `showTooltip` because we can't show for every item in the list (degrades performance, copied from old implementation)

See here: https://github.com/Expensify/App/pull/26091/files#diff-58e9afec1ac387c92300582afe20b46d7ae577a515cf97f5c4ba58948ad4fd46R247-R258

@Santhosh-Sellavel wdyt?

Reply:
Cool makes sense!

Reply:
Amazing, sent the fix